packages/airnode-validator/test/validator.test.ts

```diff
@@ -1,10 +1,24 @@
 import * as fs from 'fs';
 import path from 'path';
-import { validateWithTemplate } from '../src';
+import { validateWithTemplate, validateJsonWithTemplate } from '../src';
 
 const tests = fs.readdirSync(path.resolve(__dirname, 'validatorTests'));
 const validOutput = { valid: true, messages: [] };
 
+const secretsEnv = {
+  PROVIDER_URL: 'http://127.0.0.1:8545',
+  AIRNODE_RRP_ADDRESS: '0x0000000000000000000000000000000000000000',
+  AIRNODE_WALLET_MNEMONIC: 'achieve climb couple wait accident symbol spy blouse reduce foil echo label',
+  SS_CURRENCY_CONVERTER_API_KEY: '<enter your API key>',
+  CMC_PRO_API_KEY: '<enter your API key>',
+  HTTP_GATEWAY_API_KEY: 'apikey_example_apikey_example_apikey_example',
+  HEARTBEAT_API_KEY: '',
+  HEARTBEAT_ID: '',
+  HEARTBEAT_URL: '',
+  CHAIN_ID: '',
```

Comment:
Why are these values ok empty? Shouldn't validator complain about each of them being empty? Also, about their format? `HEARTBEAT_URL` should be a valid URL and `CHAIN_ID` should be a number, shouldn't it?

Reply:
Templates are based on docs, it was not specified that `CHAIN_ID` should be a number, but I can change it if it should.

All urls are using this regex: `^(([^:\/?#]+):)?(\/\/([^\/?#]*))?([^?#]*)$`, which basically allows anything. There was whole discussion regarding this, it was decided it should not be limiting and this one was chosen...

Reply:
The `chainId` is indeed typed as string, so this is correct (I don't see it a big deal, since I wonder how big of a convention this is and whether there exist any non number like providers).